Elements/test/LineTests.cs

```diff
@@ -902,5 +902,65 @@ public void LinesOffset_ClosedShape()
 
             Assert.Equal(3, offset.Count());
         }
+
+        [Fact]
+        public void LineDistanceSame()
```

Comment:
[Fact]
        public void LineDistanceSimple()
        {
            //Right line has 1/2 slope, so lines should meet at (6, 2, 2)
            //forming 1 by 2 triangle
            Line left = new Line((2, 0, 1), (12, 0, 1));
            Line right = new Line((6, 0, 6), (6, 3, 0));
            Assert.Equal(Math.Sqrt(5), left.DistanceTo(right));

            //Overlapping
            left = new Line((3, 3), (6, 3)); 
            right = new Line((4, 3), (5, 3));
            Assert.Equal(0, left.DistanceTo(right));
            right = new Line((4, 3), (7, 3));
            Assert.Equal(0, left.DistanceTo(right));
            right = new Line((2, 3), (5, 3));
            Assert.Equal(0, left.DistanceTo(right));

            //Collinear
            left = new Line((3, 3), (6, 3));
            right = new Line((7, 3), (9, 3));
            Assert.Equal(1, left.DistanceTo(right));

            //Parallel
            left = new Line((5, 2), (5, 8));
            right = new Line((7, 5), (7, 10));
            Assert.Equal(2, left.DistanceTo(right));

            //Intersecting
            left = new Line((5, 5), (10, 10));
            right = new Line((7, 0), (7, 10));
            Assert.Equal(0, left.DistanceTo(right));
        }

Add this test, it fails at one of the overlap cases. Group other tests under "LineDistanceParametric" so they do not take to much space in test view.

Reply:
Group all parametric tests into the same functions and remove region tags.